mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
     """
     def __init__(self, regions=["ORD"]):
         """
         Create an API with the specified regions.
         """
         self._regions = regions
+        self.session_store = None
```

Comment:
I fixed this in a commit earlier today; not sure why GIthub is still listing this as a concern.  :(